go.mod

```diff
@@ -4,6 +4,8 @@ go 1.13
 
 require (
 	github.com/pkg/errors v0.9.1
-	gopkg.in/go-ini/ini.v1 v1.52.0
-	gopkg.in/yaml.v2 v2.2.8
+	github.com/smartystreets/goconvey v1.6.4 // indirect
```

Comment:
What is this package and why are we indirectly depend on it? The description says "About Go testing in the browser." why do we need testing in the browser for a text command line utility? Is it possible to not depend on that?

Reply:
let me check the libs ...

Reply:
`go mod tidy` added this, I normaly use it to shrink go.sum file ... in this case It have added this dependencd